### PR TITLE
Created C++ Coding Standard Page

### PR DESCRIPTION
**Pull request by DannyPeck**
### Changes proposed within this pull request:
- Started the C++ Code Standard Documentation

### CODE REVIEW

**Comment by technetos** [comment]
```
if (condition)
{

}
```

File: C++/CodingStandard.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# C++ Coding Standard
+## Rules
+### 1. There will be a space before an if/for condition
+#### Correct
+```
+if (condition)
+{
```
Comment (technetos) [inline C++/CodingStandard.md]:
```
if (condition)
{

}
```

**Review by technetos — CHANGES_REQUESTED** [review]
See my comment

File: C++/CodingStandard.md (hunk 1 above)
Comment (DannyPeck) [reply C++/CodingStandard.md]: Fair enough

Comment (technetos) [reply C++/CodingStandard.md]: That was all.

**Comment by technetos** [comment]
Does this document serve as a standard for comments / doxycomments aswell?

**Comment by xmclark** [comment]
@technetos yeah seems like a good place to put that kind of stuff.

**Review by xmclark — CHANGES_REQUESTED** [review]
I've added some more content and changed the formatting.

**Comment by xmclark** [comment]
Does this need anything else? I have moved it to done.

**Comment by DannyPeck** [comment]
I say go ahead and merge it and if we need to add to it we will.

**Pull request merged**